packages/gatsby-plugin-google-tag-manager/src/index.ts

```diff
@@ -0,0 +1 @@
+// noop
```

Comment:
Why is that needed?

Reply:
I thought this would be needed if a user `require` my plugin. Not 100% sure about this https://www.gatsbyjs.com/docs/files-gatsby-looks-for-in-a-plugin/

Reply:
![image](https://user-images.githubusercontent.com/11340665/105209779-fdcfb700-5b28-11eb-81dd-4e17d2799da5.png)